fv3core/stencils/fv_dynamics.py

```diff
@@ -329,6 +329,7 @@ def compute(state, comm, timer=NullTimer()):
     if global_config.get_do_halo_exchange():
         comm.halo_update(state.phis_quantity, n_points=utils.halo)
     compute_preamble(state, comm)
+    # "remapping" loop (because there's one remapping per loop)
```

Comment:
Updated to "one remapping per iteration". The details you described are mostly documented in `do_dyn`, I wanted to avoid duplicating that information. I think it'd be great to include a more full description of what happens in the routine (or a hyperlink to the do_dyn docstring) in our final version of this routine.